### DISCUSSION TITLE
THANK YOU!!

### DISCUSSION THREAD

**Discussion by mrhhats** [GerryDazoo/Slinger#20] [General]
I have nothing else to say but THANK YOU for your hard work and dedication to make this possible =)

Grabbed the code and had it working in 10 mins!

**Comment by GerryDazoo** [comment]
Good news,  what kind of sling box do you have? Are you running the exe or the python code. If python, on what platform.

Did all your remote buttons work? There may be an issue with the current version when sending test remote keycodes. If you have any problem let me know.

 

*	Gerry

 

From: mrhhats ***@***.***> 
Sent: Friday, August 12, 2022 2:50 PM
To: GerryDazoo/Slinger ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GerryDazoo/Slinger] THANK YOU!! (Discussion #20)

 

I have nothing else to say but THANK YOU for your hard work and dedication to make this possible =)

Grabbed the code and had it working in 10 mins!
